pkgs/development/libraries/modern-cpp-kafka/gtest_envs.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
+index 4b069a6..faa328e 100644
+--- a/tests/CMakeLists.txt
++++ b/tests/CMakeLists.txt
+@@ -21,30 +21,42 @@ link_directories(${Boost_LIBRARY_DIRS})
+ #---------------------------
+ # googletest
+ #---------------------------
+-if (DEFINED ENV{GTEST_ROOT})
```

Comment:
I'm very rusty at cmake but could you instead point cmake to the gtest directory by using pkg-config as a native build input?

Reply:
Ahhhh, I see. I should have read this: `The lib expects headers and lib is in common dir, but nix splits then into out and dev derivations`. I understand now!